src/lib/InterventionList.svelte

```diff
@@ -1,18 +1,19 @@
 <script lang="ts">
+  import type { FeatureUnion } from "../types";
   import { Accordion, AccordionItem } from "carbon-components-svelte";
   import Form from "./Form.svelte";
   import {
     gjScheme,
     currentHover,
     currentlyEditing,
     openFromSidebar,
-  } from "../stores.js";
+  } from "../stores";
 
-  function interventionName(feature) {
+  function interventionName(feature: FeatureUnion): string {
     if (feature.properties.name) {
       return feature.properties.name;
     }
-    var noun = feature.properties.intervention_type;
+    var noun: string = feature.properties.intervention_type;
```

Comment:
If we don't specify this, it actually grabs the more specific type and complains when we assign other strings below